pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -448,6 +448,8 @@ func (s *ServiceImpl) buildAlertNavLinks(c *contextmodel.ReqContext) *navtree.Na
 	if hasAccess(ac.EvalAny(
 		ac.EvalPermission(ac.ActionAlertingNotificationsRead),
 		ac.EvalPermission(ac.ActionAlertingNotificationsExternalRead),
+		ac.EvalPermission(ac.ActionAlertingNotificationsTimeIntervalsRead),
```

Comment:
(So we show the Notification Policies navtree item if a user only has permissions to see Mute Timings, but not Policies)
![image](https://github.com/user-attachments/assets/6a24de13-5b2a-447b-8605-0b6ceff58b47)